fabfile/run.py

```diff
@@ -180,18 +183,19 @@ def tpch_automate(config_file='tpch_default.ini', connectionURI=''):
 
             execute(add.tpch, connectionURI= connectionURI, scale_factor=scale_factor)
             execute(tpch_queries, eval(config_parser.get(section, 'tpch_tasks_executor_types')), connectionURI,
-                    pg_version, citus_version)
+                    pg_version, citus_version, config_file)
             execute(pg.stop)
 
 
 @task
 @roles('master')
-def tpch_queries(query_info, connectionURI, pg_version, citus_version):
-    results_file = open(
-        os.path.join(config.HOME_DIRECTORY,'tpch_benchmark_results_PG-{}_Citus-{}.txt'.format(pg_version, citus_version)),
-        'a')
-    psql = '{}/bin/psql'.format(config.PG_LATEST)
-    tpch_path = '{}/tpch_2_13_0/distributed_queries/'.format(config.TESTS_REPO)
+def tpch_queries(query_info, connectionURI, pg_version, citus_version, config_file):
+    utils.mkdir_if_not_exists(config.paths[config.RESULTS_DIRECTORY])
```

Comment:
let's use https://docs.python.org/3.6/library/os.html#os.makedirs with `exists_ok` instead of writing the function ourselves (unless this a python2 project)

Reply:
this is a python2 project for now.